submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/submission/domain/service/ExtSubmissionService.kt

```diff
@@ -59,7 +59,7 @@ class ExtSubmissionService(
     ) {
         logger.info { "${sub.accNo} $user Received async submit request for ext submission ${sub.accNo}" }
         val submission = processSubmission(user, sub)
-        val (accNo, version) = submissionSubmitter.createRequest(ExtSubmitRequest(submission))
+        val (accNo, version) = submissionSubmitter.saveRequest(ExtSubmitRequest(submission))
```

Comment:
I think is not convinient to have this method named as save, the reason I choose to separate beetween create and update/save is that during creation we need to add records, update records necesary need to exists. Also in future we will need to create files records when creating only and not when saving.